src/ontology/mondo-edit.obo

```diff
@@ -326009,25 +326018,13 @@ relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/3664
 
 [Term]
 id: MONDO:0016916
-name: partial deletion of the long arm of chromosome 18
-comment: Reason: duplicate. This will be merged with MONDO:0011147 chromosome 18q deletion syndrome
-subset: disease_grouping
-subset: gard_rare
-subset: obsoletion_candidate
-subset: ordo_group_of_disorders {source="Orphanet:262146"}
-synonym: "chromosome 18q deletion" RELATED [GARD:0013000]
-synonym: "partial deletion of chromosome 18q" EXACT [Orphanet:262146]
-synonym: "partial deletion of the long arm of chromosome type 18" EXACT [MONDORULE:2, Orphanet:262146]
-synonym: "partial monosomy of chromosome 18q" EXACT [Orphanet:262146]
-synonym: "partial monosomy of the long arm of chromosome 18" EXACT [Orphanet:262146]
-xref: GARD:0013000 {source="MONDO:equivalentTo"}
-xref: ICD10:Q93.5 {source="ORDO:262146/attributed", source="ORDO:262146/ntbt", source="Orphanet:262146"}
-xref: Orphanet:262146 {source="MONDO:equivalentTo"}
-xref: UMLS:C0265472 {source="Orphanet:262146", source="MONDO:notFoundInDiseaseSubset"}
+name: obsolete partial deletion of the long arm of chromosome 18
+xref: Orphanet:262146 {source="MONDO:obsoleteEquivalent"}
 xref: UMLS:C0432443 {source="Orphanet:262146", source="MONDO:superClassOf"}
```

Comment:
To me, it doesn't make sense to have the annotation superClassOf, because this class is no longer a superClass of anything. But I don't know if we have an answer to this, it depends on https://github.com/monarch-initiative/mondo/issues/4196